src/utils/msb.nr

```diff
@@ -13,7 +15,8 @@ global MUL_DE_BRUIJN_BIT: [u64; 128] = [
         41, n1, 25, 37, n1, 47, n1, 30, 14, n1, n1, n1, n1, 22, n1, n1,
         35, 12, n1, n1, n1, 59, 42, n1, n1, 61, 3, 26, 38, 44, n1, 56
     ];
-unconstrained pub fn get_msb64(x: u64) -> u64 {
+
+unconstrained pub fn get_msb64(x: u64) -> u32 {
```

Comment:
Most of the changes in this PR are simply switching data types. This is the change most worth noting. `get_msb64` has been changed to return a `u32` as we can represent the MSB of a `u64` in a `u32`.